dvclive 1.0 (#4085)
iterative/dvc.org: content/docs/dvclive/api-reference/live/next_step.md

live = Live()

for step in range(3):
    live.log_metric("metric", 0.9)
    live.next_step()
```

## Description

DVCLive uses `step` to track the history of the metrics logged with
`Live.log_metric()`.

You can use `Live.next_step()` to increase the `step` by 1 (one).

In addition to increasing the `step` number, it will call `Live.make_report()`
and `Live.make_summary()`.

### Manual step updates

> If you want to decouple the `step` update from the rest of the calls, you can
> manually modify the `Live.step` property. This approach can be also used with
> custom `step` intervals:

Reviewer: ```suggestion
If you want custom `step` intervals or don't want to call 
`Live.make_summary()` / `Live.make_report()`, you can
manually modify the `Live.step` property:
```

Non-blocking
